stylesheets/employees.css

```diff
@@ -0,0 +1,32 @@
+.primary-btn > .default {
+    color: #f9f9f9;   
```

Comment:
Is that the same colors we have in the app theme?

Reply:
no sir

Reply:
We already have these [button colors in the app theme](https://github.com/torodeveloper/payroll-system-bellini-app/pull/1#issuecomment-2068368939). Can we use it instead?

Reply:
Yes. Is it okay if I used the #5 PR to create the changes?

The commit: [https://github.com/torodeveloper/payroll-system-bellini-app/pull/5/commits/1645e8b49543621fb21fe6b970cb3f73a3afeedb](url)

Reply:
No, sorry we need that in this PR. We have to merge this PR first.

Reply:
Done with the update.